mimseq/tRNAtools.py

```diff
@@ -363,7 +363,7 @@ def intronRemover (Intron_dict, seqIO_dict, seqIO_record, posttrans_mod_off, dou
 
 	return(seq)
 
-def modsToSNPIndex(gtRNAdb, tRNAscan_out, mitotRNAs, plastidtRNAs, modifications_table, experiment_name, out_dir, double_cca, threads, snp_tolerance = False, cluster = False, cluster_id = 0.95, posttrans_mod_off = False, pretrnas = False, local_mod = False, search='usearch'):
```

Comment:
Please remove this change - already included in [30e2bdc](https://github.com/nedialkova-lab/mim-tRNAseq/commit/30e2bdc8c338ca7fb50c4682bb20e1c4184dfb9c)

Reply:
I synced the fork with the master branch, so it's good to go now